.github/workflows/release-porting-guide.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: Ansible porting guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: Release version. For example, 11.1.0
+        required: true
+      ansible-major-version:
+        description: Major version. For example, 11
+        required: true
+
+jobs:
+  upload-porting-guide:
+    name: Extract the porting guide${{ '' }}  # Nest jobs under the same sidebar category
+    runs-on: ubuntu-latest
+    env:
+      package-url: https://files.pythonhosted.org/packages/source/a/ansible
+      ansible-tarball: ansible-${{ inputs.ansible-version }}.tar.gz
+      extracted-path: ansible-${{ inputs.ansible-version }}
+      porting-guide-rst: porting_guide_${{ inputs.ansible-major-version }}.rst
+    steps:
+    - name: Fetch and unpack the package tarball
+      run: >-
+        wget ${{ env.package-url }}/${{ env.ansible-tarball }}
+        &&
```

Comment:
Why use `>-` and `&&`, and not simply use `|` and write the commands in different lines?

If a command fails, the whole step fails anyway.

Reply:
> If a command fails, the whole step fails anyway.

@felixfontein that's actually not universally true. There are corner cases (mostly for Windows runners IIRC) where the second command would still get executed and its return code would be used for the step status. To fight this, either boilerplate like `set -eEuo pipefail` should be put in front of the script of something like `shell: bash -eEuo {0}` should be used.